<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>865</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>865</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>360</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>360</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgb(255, 255, 255);
}

QLabel{
	color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="fileLabel_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>201</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>School Report Card</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioBtnSelectAll">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>SELECT ALL LEARNERS</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioBtnSelectFew">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>SELECT LEARNERS FROM LIST</string>
      </property>
     </widget>
     <widget class="QComboBox" name="dropDownLearner">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>311</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox{
border-radius: 5px;
border: 1px solid rgb(100, 100, 100);
	background-color: rgb(255, 255, 255);
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addBtn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border: 1px solid rgb(26, 194, 161);
border-radius: 5px;
	color: rgb(26, 194, 161);
}
QPushButton:hover, QPushButton:focus{
	background-color: rgb(26, 194, 161);
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Add learner</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clrBtn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border: 1px solid rgb(100, 100, 100);
border-radius: 5px;
	color: rgb(100, 100, 100);
}
QPushButton:hover, QPushButton:focus{
	background-color: rgb(100, 100, 100);
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Clear List</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>311</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
border: 1px solid rgb(150, 150, 150);
border-radius: 5px;
}</string>
      </property>
     </widget>
     <widget class="QFrame" name="btnExport">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>480</y>
        <width>361</width>
        <height>50</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="expBtn">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	border-bottom: 2px solid rgb(17, 130, 107);
	border-radius: 5px;
	color: rgb(255, 255, 255);
	background-color: rgb(26, 194, 161);
}
QPushButton:focus{
	border-style: inset;
}</string>
         </property>
         <property name="text">
          <string>Export File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	border-bottom: 2px solid rgb(100, 100, 100);
	border-radius: 5px;
	color: rgb(255, 255, 255);
	background-color: rgb(150, 150, 150);
}
QPushButton:focus{
	border-style: inset;
}</string>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>321</width>
        <height>50</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnFilePath">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(100, 100, 100);
}
</string>
         </property>
         <property name="text">
          <string>File path</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtFilePath">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>280</width>
           <height>28</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
border-radius: 3px;
border: 1px solid rgb(170, 170, 170);
	background-color: rgb(255, 255, 255);
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../assets/excel-solid.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgb(255, 255, 255);
}

QLabel{
	color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="fileLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>201</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Template Preview</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>450</width>
        <height>410</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>410</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>450</width>
        <height>410</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
border: 1px solid rgb(150, 150, 150);
border-radius: 5px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">QLabel{
border: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../assets/cardPreview.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>480</y>
        <width>441</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>NOTE: SHS report card template has a maximum of 10 subject slots per semester</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
